python/python/lance/vector.py

```diff
@@ -134,7 +134,9 @@ def train_ivf_centroids(
     sample_rate: int = 256,
 ) -> np.ndarray:
     """Use accelerator (GPU or MPS) to train kmeans."""
-    if not (CUDA_REGEX.match(accelerator) or accelerator == "mps"):
+    if isinstance(accelerator, str) and (
```

Comment:
I guess another nit is. I think
```python
torch.device(device)
```
should work for both string and `torch.Device` inputs.